doc/release-notes/release-7.sgml.in

```diff
@@ -0,0 +1,118 @@
+<!doctype linuxdoc system>
```

Comment:
> PR branch commit 229d5582f3398f08f78071c625c056c78a4f650b message: reviewer of PR https://github.com/squid-cache/squid/pull/1268 insists it be done here

FWIW, I think _this_ PR is actually the wrong place to add `doc/release-notes/release-7.sgml.in` file. That v7-specific file should be added in #1268 instead (i.e. it should be added in the PR that migrates the master branch to future v7 work). AFAICT, I said the same thing at https://github.com/squid-cache/squid/pull/1268#discussion_r1098121227

Reply:
(I thought I posted this already, but not seeing it in github)
Simply put: There is no reason for `release-7.sgml.in` file to exist separate from the `Makefile.am` change made by this PR. Even if we were to add `release-7.sgml` from the obsolete process - this PR would be removing it and adding this new file.

Reply:
> There is no reason for release-7.sgml.in file to exist separate from the Makefile.am change made by this PR.

That is true. There is _also_ no valid reason for release-7.sgml.in file to be added in _this_ PR. Version-specific files should be added in version-specific PRs, such as PR 1268. PR 1268 should add either `release-7.sgml` or `release-7.sgml.in`, depending on which of the two PRs lands first.

> Even if we were to add release-7.sgml from the obsolete process - this PR would be removing it and adding this new file.

I disagree that this version-agnostic PR has an obligation to update/rename any already added version-specific release notes. It may do so if there is some value in doing that or postpone such out-of-scope updates to future dedicated PRs. A minimal well-scoped version-agnostic PR would not update/rename existing version-specific release notes.

We control the order in which the two PRs in question are merged, so we can, if we want to, reduce the total number of changes. However, such optimizations should still observe correct PR scope/boundaries. This is a version-agnostic PR. PR 1268 is about v7.

Reply:
Sigh. Please curb your OCD tendency @rousskov. This PR is based on the current Squid code in master branch. Simply that. It does what is necessary to have the "perfection" after it is merged.